### PR TITLE
Added possibility to tag topics with defined labels

### PR DESCRIPTION
**Pull request by kacperb333**
Allowed labels are configurable via management properties. Currently labels are displayed on topic details page.

### CODE REVIEW

**Comment by kacperb333** [comment]
Component for selecting labels:
Screenshot 2020-10-16 at 13 09 01

Multiple select supported:
Screenshot 2020-10-16 at 13 09 29

View of selected labels on topic properties:
Screenshot 2020-10-16 at 13 09 37

File: hermes-api/src/main/java/pl/allegro/tech/hermes/api/Topic.java [hunk 1]
```diff
@@ -155,7 +162,8 @@ public boolean equals(Object obj) {
                 && Objects.equals(this.maxMessageSize, other.maxMessageSize)
                 && Objects.equals(this.subscribingRestricted, other.subscribingRestricted)
                 && Objects.equals(this.publishingAuth, other.publishingAuth)
-                && Objects.equals(this.offlineStorage, other.offlineStorage);
+                && Objects.equals(this.offlineStorage, other.offlineStorage)
+                && Objects.equals(this.labels, other.labels);
```
[the remark this answers is not in the document]
Comment (kacperb333) [reply hermes-api/src/main/java/pl/allegro/tech/hermes/api/Topic.java:166]: I've missed it. Thanks 👍

File: hermes-api/src/main/java/pl/allegro/tech/hermes/api/Topic.java [hunk 2]
```diff
@@ -238,6 +246,10 @@ public TopicDataOfflineStorage getOfflineStorage() {
         return offlineStorage;
     }
 
+    public Set<TopicLabel> getLabels() {
+        return labels;
```
[the remark this answers is not in the document]
Comment (kacperb333) [reply hermes-api/src/main/java/pl/allegro/tech/hermes/api/Topic.java:250]: Good catch!

**Review by Diewa — DISMISSED** [review]

**Review by Diewa — APPROVED** [review]

**Review by mareckmareck — APPROVED** [review]

File: hermes-console/static/partials/topic.html [hunk 3]
```diff
@@ -63,6 +63,12 @@ <h3 class="panel-title">Properties</h3>
                     <p>
                         <strong>Content type:</strong> {{topic.contentType}}
                     </p>
+                    <p>
+                        <strong>Labels:</strong>
+                        <span ng-repeat="label in topic.labelValues">
```
[the remark this answers is not in the document]
Comment (kacperb333) [reply hermes-console/static/partials/topic.html:72]: I'm pretty sure you have added it in https://github.com/allegro/hermes/pull/1293/commits/a69a50c6f19d504073a21b42f309c5b51ca6fa2d

**Pull request merged**